rust/kernel/error.rs

```diff
@@ -12,6 +12,7 @@ use core::{num::TryFromIntError, str::Utf8Error};
 ///
 /// The kernel defines a set of integer generic error codes based on C and
 /// POSIX ones. These codes may have a more specific meaning in some contexts.
+#[derive(Debug)]
```

Comment:
https://github.com/Rust-for-Linux/linux/pull/180

Reply:
Could you explain why we don't want `#[derive(Debug)]` in the kernel? I used it because I want to call `expect` on `KernelResult`. BTW, since we don't run `bindgen` with option `--no-debug`, all kernel structures generated are `#[dervice(Debug)]`

Reply:
Ah, yes, that is fair. I forgot it uses the debugging message.

I was thinking more about having `Debug` just for no reason -- i.e. just because somebody used it in some temporary `{:?}` that got removed later etc.